pkg/kube/kube_nodes.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package kube
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/ethersphere/node-funder/pkg/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+	"log"
+	"math/big"
+	"net/http"
+	"os"
+	"sync"
+)
+
+const (
+	beeAddressEndpoint = "/addresses"
+	beeWalletEndpoint  = "/wallet"
+)
+
+type NamespaceNodes struct {
+	Name  string
+	Nodes []Node
+}
+type Node struct {
+	Name      string
+	Ip        string
+	BeeTokens BeeTokens
+}
+
+type BeeTokens struct {
+	EthAddress      string
+	ChainID         int
+	ContractAddress string
+	ETH             *big.Int
+	BZZ             *big.Int
+	GBZZ            *big.Int
+}
+
+func NewKube() (*corev1client.CoreV1Client, error) {
+	config, err := GetConfig()
+	if err != nil {
+		log.Printf("get configuration failed with error: %v", err)
+		return nil, err
+	}
```

Comment:
Instead of logging error here it would be better to return wrapped error since caller would have more information why call has failed.

```
if err != nil {
		return nil, ftm.Errorf("geting k8s configuration failed: %w, err)
	}
```